support-files/mysql.spec.sh

```diff
@@ -246,7 +180,7 @@
 %define license_files_server    %{src_dir}/LICENSE.mysql
 %define license_type            Commercial
 %else
-%define license_files_server    %{src_dir}/COPYING %{src_dir}/README
+%define license_files_server    COPYING README
```

Comment:
Hi Teemu!


On 22.01.2015 12:29, Teemu Ollakka wrote:
>> @@ -246,7 +180,7 @@
>>   %define license_files_server    %{src_dir}/LICENSE.mysql
>>   %define license_type            Commercial
>>   %else
>> -%define license_files_server    %{src_dir}/COPYING %{src_dir}/README
>> +%define license_files_server    COPYING README
>
> This had to be changed due to change in %setup directive, those files were not found from the specified locations at some point. Need to check if it was a temporary issue.

I noticed the change in "%setup", and I didn't see a reason.
What was your reason for it?

Unless there is some good reason, IMHO we should do neither that 
"%setup" change nor any of the consequential ones - the more we change, 
the greater our effort will be when we have to follow Oracle development.

>
> Anyway, is there a difference between files found from under %{src_dir} and under build dir?

Technically, I'm not aware of any difference.
Having "%{src_dir}" as a separate directory provides a kind of 
documentation which files come from the source tarball and which ones 
are generated during build (unless the scriptlets take explicit action).
IMO, this makes understanding easier.


Regards,
Jörg

-- 
Joerg Bruehe, Senior MySQL Support Engineer, joerg.bruehe@fromdual.com
FromDual GmbH, Rebenweg 6, CH - 8610 Uster
Geschäftsführer: Oliver Sennhauser
Handelsregister-Eintrag: CH-020.4.044.539-3